org.eevolution.warehouse/src/main/java/base/org/spin/form/OutBoundOrder.java

```diff
@@ -1131,9 +1360,9 @@ public String validStock(IMiniTable stockTable) {
 				.append(product)
 				.append("[")
 				.append("@M_Warehouse_ID@=").append(warehouse)
-				.append(" @QtyAvailable@=").append(m_QtyOnHand.subtract(m_QtyInTransit).doubleValue())
-				.append(" @QtyToDeliver@=").append(m_QtySet.doubleValue())
-				.append(" @PickedQty@=").append(m_QtyAvailable.doubleValue())
+				.append(" @QtyAvailable@=").append(qtyOnHand.subtract(qtyInTransit).doubleValue())
```

Comment:
Please make this code more atomic, the refactory can be change in other pull request as Source code Improve